packages/documentation/src/pages/getting-started/bundles-and-code-organization.mdx

```diff
@@ -0,0 +1,86 @@
+# Bundle and Code Organization
+
+## Files Overview
+
+- `/config`
+  - jest configs - proof of concepts- not used in CI
+    - `jest-integration.config.js`
+    - `jest-vrt.config.js`
+  - Nightwatch e2e test runner and configurations
+    - `nightwatch-globals-sauce.js`
+    - `nightwatch-sauce.js`
+    - `nightwatch.docker-compose.js`
+    - `nightwatch.js`
+  - sass config: `sass-lint.yml`
+  - webpack production and dev configs
+    - `webpack.config.js`
+    - `webpack.dev.config.js`
+  - build setup script
+    - `manifest-loader.js` - passes front matter of content files to app entry files
+	  - `parse-app-settings.js` - reads settings.js - a file containing app information generated during Metalsmith build
+- `/docs` - best practices and troubleshooting docs
+- `/jenkins` - jenkins setup, build, deploy script for `vets-websites` repo
+- `/script`
+  - `/custom-eslint` - project specific linting plugin
+  - `/hooks` - -git -precommit hook
+  - `app-list.sh` - bash script that outputs a list of the applications in vets-website
+  - `build.js` - imports and runs build script in src/site/stages/build
+  - `csv-path-builder.js` - TODO
+  - `heroku-postbuild.sh` - TODO - content deploy to heroku
+  - `mocha.js` - unit test runner and watch task
+  - `prearchive.js` - imports and run prearchive in src/site/stages/prearchive TODO
+  - `prebuild.js` - npm prebuild script that checks node version and install precommit hooks
+  - `preview.js` - metalsmith build script for drupal preview
+  - `reset-environment.js` - installs yarn and cleans up node_modules- used by build process
+  - `run-coverage.sh` - runs mocha unit test coverage report
+  - `run-docker-nightwatch.sh` - runs mock vets-api with docker
+  - `run-mockapi.sh` - runs js mock vets-app
+  - `run-nightwatch.sh` - test harness for running end to end tests
+  - `run-puppeteer-tests.sh` - test harness for puppeteer tests (proof of concept)
+  - `run-unit-test.sh` - runs unit tests
+  - `security-check.js` - checks for security advisories in dependencies
+  - `update-json-schema.js` - helper for updating dependency version of vets-jason-schema
+  - `watch.js` - watch task for unit tests (proof of concept)
+- `/src` - source files for vets-website web applications
+  - `/applications` - application specific code that's into -entry.js files
+  - `/platform` - shared code that's bundled into vendor.js, injected directly into the site, or used for testing
+  - `/site` - build specific code including templates used for translating content from drupal CMS for the metalsmith build process
```

Comment:
Maybe "Metalsmith pipeline code and templates used to generate static content pages from Drupal api or Markdown files"